packages/analytics/src/events.ts

```diff
@@ -102,3 +106,28 @@ interface HistoricalEvents {
 }
 
 export type EventName = keyof Events;
+
+type UserSettingValue =
+  | { account_display: 'stacks' | 'native-segwit' | 'taproot' | 'bns' }
+  | { analytics: 'consent-given' | 'rejects-tracking' }
+  | { bitcoin_unit: 'bitcoin' | 'satoshi' }
+  | { email_address: string }
+  | { fiat_currency: string }
+  | {
+      network:
+        | 'mainnet'
+        | 'testnet'
+        | 'testnet4'
+        | 'signet'
+        | 'sbtcTestnet'
+        | 'sbtcDevenv'
+        | 'devnet';
```

Comment:
We could pull values from models such that this'll update when we add new networks

Reply:
Done